bundles/org.openhab.binding.jellyfin/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="jellyfin"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="server">
+		<label>Jellyfin Server</label>
+		<description>Represents a running Jellyfin server instance</description>
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname/IP</label>
+				<description>Hostname or IP address of the server</description>
+				<default>127.0.0.1</default>
+			</parameter>
+			<parameter name="port" type="integer" min="0" max="65535" required="true">
+				<label>Port</label>
+				<description>Port of the server</description>
+				<default>8096</default>
+			</parameter>
+			<parameter name="ssl" type="boolean" required="true">
+				<label>SSL</label>
+				<description>Connect through https</description>
+				<default>false</default>
+			</parameter>
+			<parameter name="refreshSeconds" type="integer" min="10" max="300" required="true">
+				<label>Refresh Seconds</label>
+				<description>Interval to pull devices state from the server</description>
+				<default>30</default>
+			</parameter>
+			<parameter name="clientActiveWithInSeconds" type="integer" min="0" max="950" required="true">
+				<label>Client Active With In Seconds</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions